src/pages/references/stacks-node-configuration.md

```diff
@@ -340,6 +340,36 @@ Example:
 walk_interval = 30
 ```
 
+#### read_only_call_limit_read_length
+
+Total amount of data in bytes allowed to be retrieved from a read-only smart-contract call.
+
+Example:
+
+```toml
+read_only_call_limit_read_length = 100000
+```
+
+#### read_only_call_limit_read_count
+
+Total amount of data pieces allowed to be retrieved from a read-only smart-contract call.
+
+Example:
+
+```toml
+read_only_call_limit_read_count = 30
+```
+
+#### read_only_call_limit_runtime
+
+Total execution cost limit for executing read-only smart-contracts.
+
+Example:
+
+```toml
+read_only_call_limit_runtime = 1000000000
+```
+
```

Comment:
@lgalabru or @reedrosenbluth  Do these descriptions look correct to you? Also, what units is `read_only_call_limit_runtime` in?

Reply:
Left some comments above.

Runtime cost is a unitless metric. It loosely represents CPU cycles and exists as a basis for comparison. Learn more here: https://github.com/stacksgov/sips/blob/2d3fd9bf8da7a04f588d90ff6252173d7609d7bf/sips/sip-006/sip-006-runtime-cost-assessment.md#introduction